commons/all/src/main/java/org/infinispan/commons/util/TypedProperties.java

```diff
@@ -88,14 +88,10 @@ public long getLongProperty(String key, long defaultValue, boolean doStringRepla
       }
    }
 
-   public long getDurationProperty(String key, long defaultValue) {
-      return getDurationProperty(key, defaultValue, false);
-   }
-
    public long getDurationProperty(String key, long defaultValue, boolean doStringReplace) {
```

Comment:
`getIntProperty` and `getLongProperty` require the same fix.

Reply:
are you sure about this ? The thing with those is that the notion of "duration" corresponds to Int and Long, but int value or long value are only applied in LONG and INT properties

Reply:
In my opinion, yes. It is just a simple fix, and I wouldn't be surprised if a user puts an int/short/byte in the properties expecting it to work. 
It is up to you. I'll merge the PR either way.